### ISSUE TITLE
Fix violations in a11y report for docs site homepage

### ISSUE THREAD

**Issue by thyhjwb6** [Royal-Navy/design-system#987]
When running `axe-core` against the docs site homepage, a11y violations were found which need fixing.

## Violations
Issue|Element
-- | --
Ensures the   contrast between foreground and background colors meets WCAG 2 AA contrast   ratio thresholds | \<div   class="rn-container">There's a new version of the Royal Navy   Design System.\<strong>Read the migration   docs.</strong></div>
Ensures the contrast between foreground   and background colors meets WCAG 2 AA contrast ratio thresholds | \<strong>Read the migration docs.</strong>
Ensures the contrast between foreground   and background colors meets WCAG 2 AA contrast ratio thresholds | \<span class="rn-badge rn-badge--action rn-badge--solid   rn-badge--regular rn-badge--undefined">v2.8.0-alpha.6</span>
Ensures the contrast between foreground   and background colors meets WCAG 2 AA contrast ratio thresholds | \<span class="rn-badge rn-badge--action rn-badge--solid   rn-badge--small rn-badge--undefined">alpha</span>
Ensures every id attribute value is   unique | \<path id="arrow-right-icon_svg__a" d="M11.841   17.149l5.74-5.57c.25-.24.374-.569.374-.93v-.017c0-.362-.125-.69-.374-.931l-5.74-5.57c-.499-.519-1.3-.519-1.795   0a1.375 1.375 0 000 1.88l3.45 3.3H4.6c-.707 0-1.276.594-1.276 1.33 0 .747.57   1.33 1.272 1.33h8.894l-3.45 3.3a1.375 1.375 0 000 1.878c.499.52 1.301.52 1.8   0z">
Ensures every id attribute value is   unique | \<mask id="arrow-right-icon_svg__b"   fill="#fff">\<use   xlink:href="#arrow-right-icon_svg__a"></use></mask>
Ensures links have discernible text | \<a href="/">
Ensures links have discernible text | \<a href="https://github.com/Royal-Navy/design-system"   target="_blank" rel="noopener noreferrer">
Ensures all page content is contained by   landmarks | \<html>

## Acceptance criteria
- [ ] Fix all violations
- [ ] Rerun check

**Comment by hxltrhuxze** [comment]
Closing as this work will be superseded by the new documentation site.
